extensions/wikia/FeedsAndPosts/FeedsAndPostsController.class.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+use Wikia\FeedsAndPosts\JustUpdatedAPIProxy;
+
+class FeedsAndPostsController extends WikiaController {
+	public function getJustUpdated() {
```

Comment:
where this data is used? Is there any template? or is it called from other front-end apps?

Reply:
Fepo will use it